filewriter_v2/README.md

```diff
@@ -0,0 +1,388 @@
+
+| Version | Configuration | NeXus/NXmx Version | Multi-channel data support |
+|----|----|----|----|
+| V1 | SIMPLON | loosely based on [NXmx 1.4](https://github.com/nexusformat/definitions/releases/tag/NXmx-1.4) | no |
+| V2 | SIMPLON | [NeXus 2022.07 NXmx](https://github.com/nexusformat/definitions/blob/v2022.07/applications/NXmx.nxdl.xml) | yes |
+
+[^1]: This is not part of the NeXus standard yet, a detailled PR will be opened when this data format has been approved by the community. The gerenal idea of this representation has already been discussed and approved by the NIAC though, see this [issue](https://github.com/nexusformat/definitions/issues/940).
+
+Therefore, a FileWriter V2 file is structured in the following groups:
+
+```
+/
+|
++-- entry
+    |
+    +-- data
+    +-- instrument
+    |   |
+    |   +-- beam
+    |   +-- detector
+    |       |
+    |       +-- threshold_1_channel*
+    |       +-- threshold_2_channel*
+    |       +-- difference_channel*
+    |       +-- geometry
+    |       +-- module
+    |       +-- transformations
+    |
+    +-- sample
+    |   |
+    |   +-- transformations
+    |   +-- beam
+    |
+    +-- source
+```
+\* this is not part of the NeXus standard yet, see [Multi-Channel Data](#multi-channel-data).
+
+## FileWriter V2 Configuration
+
+FileWriter V2 can be enabled and configured using the SIMPLON API.
+
+### SIMPLON FileWriter Configuration Parameters
+
+The configuration interface is accessible at `http://<ADDRESS_OF_DCU>/filewriter/api/1.8.0/config`.
+
+| Parameter | JSON Type | Default | Description |
+|-----------|------|---------|-------------|
+| `mode` | string | `disabled` | Enabled state of the FileWriter, either `enabled` or `disabled`. |
+| `compression_enabled` | bool | `true` | Optionally disable compression of image data. |
+| `image_nr_start` | uint | `1` | First image number on the axis `/entry/image_number`. |
+| `name_pattern` | string | `series_$id` | Basename of the files. `$id` will be replaced by the series number, i.e. using the default, the generated files will have the names:<br> `series_<series_number>_master.h5`<br> `series_<series_number>_data_<file_number>.h5`|
+| `nimages_per_file` | uint | `1000` | Maximum number of images stored in each data file. If set to `0`, all images are stored directly in the master file and no data files are created. If set to a value greater than zero, the images are stored in multiple data files. |
+| `v2_preview_enabled` | bool | `false` | Enabled state of the FileWriter V2 preview. |
```

Comment:
Should we additionally write that this will change? I mean the "preview" part makes it kinda obvious that it will change, but still, make this explicit. Might even be a different key altogether such as "format"

Reply:
I can add an extra warning, sure.

Reply:
```suggestion
| `v2_preview_enabled` | bool | `false` | Enabled state of the FileWriter V2 preview. <br>**NOTE:** This key will be replaced when FileWriter V2 is released. It is likely that there will be a `format` key instead, in analogy to Stream V2. |
```

Reply:
How do you feel about that suggestion, @diego-gaemperle ?

Reply:
I would rephrase the second sentence to:
"There will likely be..."
Otherwise sounds good to me

Reply:
Done in [2e366ce](https://github.com/soph-dec/documentation/pull/1/commits/2e366ceea5bace74784d3ceab0b02babd786a421)